yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1258,10 +1258,17 @@ object Client extends Logging {
         logDebug("Attempting to fetch HBase security token.")
 
         val hbaseConf = confCreate.invoke(null, conf)
-        val token = obtainToken.invoke(null, hbaseConf).asInstanceOf[Token[TokenIdentifier]]
-        credentials.addToken(token.getService, token)
-
-        logInfo("Added HBase security token to credentials.")
+        val hbaseConfGet = (param: String) => Option(confClass
+          .getMethod("get", classOf[java.lang.String])
+          .invoke(hbaseConf, param))
+        val zkQuorum = hbaseConfGet("hbase.zookeeper.quorum")
```

Comment:
Any particular reason to the zookeeper.quorum conf here?  I'm guessing you just chose one to try.